changelogs/fragments/1813-implement_max_rc_mvs_raw.yml

```
@@ -0,0 +1,6 @@
+minor_changes:
+  - zos_mvs_raw - Add max_rc parameter. Now when the user set max_rc the module will return
```

In ansible modules, parameters are named options.

```suggestion
  - zos_mvs_raw - Added ``max_rc`` option. Now when the user sets ``max_rc``, the module tolerate the failure
     if the return code is smaller than the ``max_rc`` specified, however, return value ``changed`` will be False if
     the program return code is not 0.
```